docs/DeveloperGuide.md

```diff
@@ -510,11 +508,11 @@ specified otherwise)
     
     Use case resumes at step 1.
 
-**Use case: `US03 - Delete a patient`**
+**Use case: `UC03 - Delete a patient`**
 
 **MSS**
 
-1.  User requests to view patient records(UC01).
+1.  User requests to _view patient records(UC01)_.
```

Comment:
Should the other lines be formatted the same as well? Cause it only seems to be on one line.

Reply:
The underline is when referencing other uses cases. The other use cases referenced are being highlighted already.